socorro/unittest/external/postgresql/test_base.py

```diff
@@ -93,7 +91,13 @@ def setUp(self):
         super(IntegrationTestBase, self).setUp()
 
         cursor = self.connection.cursor()
-
+        # NOTE(willkg): Sometimes, there are items in the reports and this causes test_counts to
+        # fail. This truncates it first. It'd be nice to fix whatever is causing this problem,
+        # though.
+        cursor.execute("""
+            TRUNCATE reports CASCADE;
+        """)
+        self.connection.commit()
```

Comment:
This test fails for me periodically and it's irritating. I hit failures when I was working on this PR. Figured I'd just throw this in here and a note.